src/app/qgsmaptoolmovefeature.cpp

```diff
@@ -200,7 +200,9 @@ void QgsMapToolMoveFeature::cadCanvasReleaseEvent( QgsMapMouseEvent* e )
         QgsGeometry geom = f.geometry();
         geom.translate( dx, dy );
         f.setGeometry( geom );
+#ifdef QGISDEBUG
         const QgsFeatureId  fid = f.id();
+#endif
```

Comment:
Just update the QgsDebugMsg to diretly use `f.id()`, I think that will look cleaner.